docs/gno-docs/docs/getting-started/working-with-key-pairs.md

```diff
@@ -0,0 +1,190 @@
+---
+id: working-with-key-pairs
+---
+
+# Working with Key Pairs
+
+## Overview
+
+In this tutorial, you will learn how to manage private user keys, which are required for interacting with the Gno.land
+blockchain. You will understand what mnemonics are, how they are used, and how you can make interaction seamless with
+Gno.
+
+## Prerequisites
+
+- **`gnokey` set up. Reference the [Installation](installation.md#3-installing-other-gno-tools) guide for steps**
+
+## Listing available keys
+
+`gnokey` works by creating a local directory in the filesystem for storing (encrypted!) user private keys.
+
+You can find this repository by checking the value of the `--home` flag when running the following command:
+
+```bash
+gnokey --help
+```
+
+Example output:
+
+```bash
+USAGE
+  <subcommand> [flags] [<arg>...]
+
+Manages private keys for the node
+
+SUBCOMMANDS
+  add        Adds key to the keybase
+  delete     Deletes a key from the keybase
+  generate   Generates a bip39 mnemonic
+  export     Exports private key armor
+  import     Imports encrypted private key armor
+  list       Lists all keys in the keybase
+  sign       Signs the document
+  verify     Verifies the document signature
+  query      Makes an ABCI query
+  broadcast  Broadcasts a signed document
+  maketx     Composes a tx document to sign
+
+FLAGS
+  -config ...                                          config file (optional)
+  -home /Users/zmilos/Library/Application Support/gno  home directory
+  -insecure-password-stdin=false                       WARNING! take password from stdin
+  -quiet=false                                         suppress output during execution
+  -remote 127.0.0.1:26657                              remote node URL
+
+error parsing commandline arguments: flag: help requested
+```
+
+In this example, the directory where `gnokey` will store working data
+is `/Users/zmilos/Library/Application Support/gno`.
```

Comment:
Did you mean to check-in your personal username? We can always put in a placeholder value like <USER> here or something otherwise.